lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -116,10 +116,29 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
     if (fci) v5andv6metrics.push(fci);
     if (fmp) v5andv6metrics.push(fmp);
 
+    /** @type {Record<string, string>} */
+    const acronymMapping = {
+      'cumulative-layout-shift': 'CLS',
+      'first-contentful-paint': 'FCP',
+      'first-cpu-idle': 'FCI',
+      'first-meaningful-paint': 'FMP',
+      'interactive': 'TTI',
+      'largest-contentful-paint': 'LCP',
+      'speed-index': 'SI',
+      'total-blocking-time': 'TBT',
+    };
+
+    /**
+     * Clamp figure to 2 decimal places
+     * @param {number} val
+     * @return {number}
+     */
+    const clampTo2Decimals = val => Math.round(val * 100) / 100;
```

Comment:
feels like we should just completely round everything but CLS

Reply:
ok. wonder what the chances are of this rounding resulting in an off-by-1 perf score :p 

/nerdsnipe @brendankenny

Reply:
lol @brendankenny is this what kicked off #13316